src/components/Input/index.tsx

```diff
@@ -7,6 +7,14 @@ import styles from './Input.module.scss';
 export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   /** The default color of the input border */
   outlineColor?: Colors;
+  /** Sets the background to be translucent (2023 website figma) */
+  translucentBackground?: boolean,
```

Comment:
Can you make this a configurable option like translucentPercent or something, similar to weight for Typography: https://github.com/hack-the-6ix/react-ui/blob/2023/src/components/Typography/index.tsx

Reply:
But maybe make it a percent option that is converted to the decimal  for the css variable for ease of use (eg. 0-100 instead of 0-1 by dividing the prop before setting the css variable)